orchagent/main.cpp

```diff
@@ -397,6 +401,9 @@ int main(int argc, char **argv)
                 sairedis_rec_filename = optarg;
             }
             break;
+        case 'k':
+            gMaxBulkSize = atoi(optarg);
```

Comment:
>gMaxBulkSize [](http://example.com/codeflow?start=12&length=12)

Please handle error and corner cases.

Reply:
Added a logic to handle errors and corner cases.